packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -165,13 +165,19 @@ class _ToolbarContainerLayout extends SingleChildLayoutDelegate {
 ///  * [Scaffold], which displays the [AppBar] in its [Scaffold.appBar] slot.
 ///  * [SliverAppBar], which uses [AppBar] to provide a flexible app bar that
 ///    can be used in a [CustomScrollView].
+///    * Cookbook: [Place a floating app bar above a list](https://flutter.dev/docs/cookbook/lists/floating-app-bar)
```

Comment:
nit: indentation.

Reply:
This is intended. The cookbook article talks about floating bar, so I added it as a sub-node to the SliverAppBar list item. If that's too confusing or if we try to stay away from nested lists in API docs, I can correct this (and the other instance below).

Reply:
I would prefer not nesting these to keep them simpler.